how-we-work/mozdotorgdurable.md

```diff
@@ -1,62 +1,118 @@
+    ├── Project #2 Board
+    ├── Project #3 Board
+    └── Project #4 Board
+```
+
+### Project Board & Projects
+
+Significant projects are tracked on our [Project Board](https://github.com/mozilla/bedrock/projects/3).
+
+Rules for this board are as follows:
+
+1. Any significant project (defined as work containing multiple tasks with multiple people) must have a card on the board.
+1. The project on the board must have a description containing links to any background information and that projects task board. 
+1. Keep the contents of these cards limited, discussion about actual work should occur within those tasks.
+1. Only the following labels can be applied to projects:
+      1. The `Project` label.
+      2. The correct `Team` label. One of the following: Retention, Consumers, Firefox Growth, or Moz.org . 
+      3. Blocked if blocked.
+1. No other labels should be applied to projects.
+1. Folks who are working on a project, should be assigned.
+
+#### Project Board Columns
+
+- `Icebox` - Projects that have been prioritized to occur next.
+- `Ready` - Prioritized projects that are ready to start.
+- `In Progress` - Projects that are currently underway.
+- `Done` - Shipped!
+
+#### Moz.org Backlog & Small Tasks
+
+For non project work or tasks too small for a project, just file an issue. The lifecycle of non-project work is as follows:
+
+1. An issue is filed.
+1. If the issue is ready to work on the label "ready" is applied.
+1. If urgent, the PM or Team Lead moves it to the Project Board and either assigns someone or someone volunteers.
+1. If not urgent, the work can be picked up whenever someone needs a break or has slack time.
+1. Once moved to the board remove the "ready" label. 
+1. If there is more than one task use a note with references to keep this tidy (see below).
+1. Move the card appropriately as work progresses.
+
+![Github References](/mozdotorgdurable/references.png)
+
+### Project Boards
+
+Any significant project should have it's own project board where work is managed. When work on a project is complete, the board can be closed. Nice job!
 
 ## Board Details
 
-* What are your columns, what do the columns mean?
+### What are your columns, what do the columns mean?
+
+Generally our projects will have the following columns:
+
+- `Backlog` - Tasks we know have to get done but are not necessarily ready to go.
+- `Ready` - Prioritized tasks that are ready to start.
+- `In Progress` - Tasks that are currently underway.
+- `Done` - Shipped!
 
-Board columns:  Incoming | Research | Ready | In progress | Review | Done | Archived 
+### How/when do cards move across the board?
 
-* How/when to cards move across the board?
+Cards are moved by any member of the team when a change in status is identified. Cards should generally be moved by whomever takes the work or causes an update to status. This is not the responsibility of the Program Manager, team members show own the cards they take on including keeping the status up to date.
```

Comment:
I'm not following this sentence:

> This is not the responsibility of the Program Manager, team members show own the cards they take on including keeping the status up to date.

I believe it should be 2 sentences if I'm getting the proper meaning. e.g.:

> Moving cards is not the responsibility of the Program Manager. The team members own the cards they take including keeping the status up to date.